### ISSUE TITLE
Does it support ssr1 devices?

### ISSUE THREAD

**Issue by ZEGEIZI** [jcfain/TCodeESP32#19]
Does this project not support ssr1 devices? After flashing this firmware, I am unable to control the device.

**Comment by jcfain** [comment]
Yes the BLDC build is currently only for SSR1 as its the only TCode device currently that uses BLDC motors.
